_posts/en/newsletters/2025-05-30-newsletter.md

```diff
@@ -0,0 +1,190 @@
+  network adopts [attributable failures][topic attributable failures],
+  particularly telling the spender the amount of time it took to forward
+  a payment at each hop.  Citing several papers, she describes two types
+  of deanonymization attacks:
+
+  - An attacker operating one ore more forwarding nodes who uses timing
+    data to determine the number of hops used by a payment ([HTLC][topic
+    htlc]), which can be combined with knowledge of the topography of
+    the public network to narrow the set of nodes that might have been
+    the receiver.
+
+  - An attacker who uses an IP network traffic forwarder
+    ([autonomous system][]) to passively monitor traffic and combines
+    that with knowledge of the IP network latency between nodes (i.e.,
+    their ping times) plus knowledge of the topography (and other
+    characteristics) of the public Lightning Network.
+
+  She then describes possible solutions, including:
+
+  - Encouraging receivers to delay acceptance of an HTLC by a small
+    random amount to prevent timing attacks attempting to identify the
+    receiver's node.
+
+  - Encouraging spenders to delay resending failed payments (or
+    [MPP][topic multipath payments] parts) by a small random amount and
+    by using alternative paths to prevent timing and failure attacks
+    attempting to identify the spender's node.
+
+  - Increased payment splitting with MPP to make it harder to guess the
+    amount being spent.
+
+  - Allowing spenders to opt-in to having their payments forwarded less
+    quickly, as previously proposed (see [Newsletter #208][news208
+    slowln]).  This could be combined with HTLC batching, which is
+    already implemented in LND (although the addition of a randomized
+    delay could make it more privacy-improving).
+
+  - Reducing the precision of attributable failure timestamps to avoid
+    penalizing forwarding nodes that add small random delays.
+
+  Discussion from multiple participants evaluated the concerns and
+  proposed solutions in more detail, as well as discussing other
+  possible attacks and mitigations.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Which transactions get into blockreconstructionextratxn?]({{bse}}116519)
+  Glozow explains how the extrapool data structure (see [Newsletter #339][news339
+  extrapool]) caches rejected and replaced transactions seen by the node
+  and explains the criteria for exclusion and eviction.
```

Comment:
```suggestion
  and lists the criteria for exclusion and eviction.
```
(or maybe "gives")
Otherwise there are two "explains" in the same sentence, which seems to over-explain somewhat. :)